app/gui/playwright.config.ts

```diff
@@ -10,7 +10,7 @@ import { defineConfig } from '@playwright/test'
 import net from 'net'
 
 const DEBUG = process.env.DEBUG_E2E === 'true'
-const TIMEOUT_MS = DEBUG ? 100_000_000 : 30_000
+const TIMEOUT_MS = DEBUG ? 100_000_000 : 60_000
```

Comment:
Increasing the timeout significantly affect the test pass time (if we have a failing test, we'll have to wait for `60_000 * 5 retiries` instead of `30_000 * 5 retries` before test fail (2.5mins longer for each fail!). This will also affect the DX because if the test fails, we have to wait for longer to see the result (each iteration becomes much slower). Do we really need that?

Reply:
yeah i *think* it does, i'm not a fan of it either but it's certainly causing *some* flakiness... perhaps we should only raise the timeout on CI? because i magine CI has slower machines - personally i've never (or rarely) experienced timeouts on my local machine - and CI machines *shouldn't* have timeouts because (ideally) you're supposed to run the E2E tests locally before push